astropy/_erfa/setup_package.py

```diff
@@ -31,9 +32,22 @@ def pre_build_ext_hook(cmd_obj):
 def pre_sdist_hook(cmd_obj):
     preprocess_source()
 
+def is_release():
+    if sys.version_info[0] < 3:
+        # in py 2.x this import works fine
+        from .. import version
+        return version.release
+    else:
+        # in py 3.x the import fails because _erfa hasn't been imported yet.
+        # this hack instead finds the actual version.py file and executes it.
+        path_to_vers = os.path.join(os.path.dirname(__file__), '..', 'version.py')
+        dct = {'__file__': os.path.abspath(path_to_vers),
+               '__name__': 'astropy.version'}
+        with open(dct['__file__']) as f:
+            exec(f.read(), dct)
```

Comment:
It was very hacky... simply because I didn't know about the importlib trick!  Thanks for pointing that out - just pushed up a new version implimenting this approach.